css/styles.css

```diff
@@ -113,3 +152,77 @@ header, footer {
 .tt-cursor, .tt-suggestion:hover {
     background-color: #efefef;
 }
+
+.zen-logo {
+    width: 100px;
+    height: 100px;
+    display: block;
+    overflow: hidden;
+    border-radius: 6px;
+    margin: 0 auto;
+}
+
+.btn-search {
+    width: 50px;
+}
+
+@media all and (min-width: 480px) {
+
+    .input-search {
+        height: 50px;
+        font-size: 18px !important;
+    }
+
+    .btn-search {
+        height: 50px;
+    }
+
+    .fa-search {
+        font-size: 20px;
+    }
+
+}
+
+.sponsor {
+    width: 70%;
+    margin: 30px auto 0 auto;
+}
+
+.sponsor-a {
+    width: 100%;
+    display: block;
+    margin-top: 20px;
+}
+
+.sponsor-a:first-of-type {
+    margin-top: 0;
+}
+
+@media all and (min-width: 640px) {
+
+    .sponsor-a {
+        width: 30.666666667%;
+        float: left;
+        margin-left: 4%;
+        margin-top: 0;
+    }
+
+    .sponsor-a:first-of-type {
+        margin: 0;
+    }
+
+}
+
+.color-grey {
+    color: #666666 !important;
+}
+
+.clear-both {
+    clear: both;
+    height: 0px;
```

Comment:
Sorry, I was out all day... Yeah, it could be without the `px`. I will do that now